android/assets/jsons/translations/English.properties

```diff
@@ -5292,6 +5292,123 @@ Coevorden =
+Oh, welcome! = 
+ # Requires translation!
+Oh, it is you. = 
+ # Requires translation!
+Nobel Prize = 
+ # Requires translation!
+Gain [amount] [param] with a [param2] gift to a City-State = 
+ # Requires translation!
+When declaring [param], both parties gain a +[amount]% boost to [param2] = 
+ # Requires translation!
+All hail the transcendent King Gustavus Adolphus, founder of the Swedish Empire and her most distinguished military tactician. It was during your reign that Sweden emerged as one of the greatest powers in Europe, due in no small part to your wisdom, both on and off the battlefield. As king, you initiated a number of domestic reforms that ensured the economic stability and prosperity of your people. As the general who came to be known as the "Lion of the North," your visionary designs in warfare gained the admiration of military commanders the world over. Thanks to your triumphs in the Thirty Years' War, you were assured a legacy as one of history's greatest generals. = 
+ # Requires translation!
+Oh noble King, the people long for your prudent leadership, hopeful that once again they will see your kingdom rise to glory. Will you devise daring new strategies, leading your armies to victory on the theater of war? Will you build a civilization that stands the test of time? = 
+ # Requires translation!
+Stockholm = 
+ # Requires translation!
+Uppsala = 
+ # Requires translation!
+Gothenburg = 
+ # Requires translation!
+Malmö = 
+ # Requires translation!
+Linköping = 
+ # Requires translation!
+Kalmar = 
+ # Requires translation!
+Skara = 
+ # Requires translation!
+Västerås = 
+ # Requires translation!
+Jönköping = 
+ # Requires translation!
+Visby = 
+ # Requires translation!
+Falun = 
+ # Requires translation!
+Norrköping = 
+ # Requires translation!
+Gävle = 
+ # Requires translation!
+Halmstad = 
+ # Requires translation!
+Karlskrona = 
+ # Requires translation!
+Hudiksvall = 
+ # Requires translation!
+Örebro = 
+ # Requires translation!
+Umeå = 
+ # Requires translation!
+Karlstad = 
+ # Requires translation!
+Helsingborg = 
+ # Requires translation!
+Härnösand = 
+ # Requires translation!
+Vadstena = 
+ # Requires translation!
+Lund = 
+ # Requires translation!
+Västervik = 
+ # Requires translation!
+Enköping = 
+ # Requires translation!
+Skövde = 
+ # Requires translation!
+Eskilstuna = 
+ # Requires translation!
+Luleå = 
+ # Requires translation!
+Lidköping = 
+ # Requires translation!
+Södertälje = 
+ # Requires translation!
+Mariestad = 
+ # Requires translation!
+Östersund = 
+ # Requires translation!
+Borås = 
+ # Requires translation!
+Sundsvall = 
+ # Requires translation!
+Vimmerby = 
+ # Requires translation!
+Köping = 
+ # Requires translation!
+Mora = 
+ # Requires translation!
+Arboga = 
+ # Requires translation!
+Växjö = 
+ # Requires translation!
+Gränna = 
+ # Requires translation!
+Kiruna = 
+ # Requires translation!
+Borgholm = 
+ # Requires translation!
+Strängnäs = 
+ # Requires translation!
+Sveg = 
+
```

Comment:
It looks like you did this manually, this is not necessary. Before each update, Yairm regenerates all these files automatically, so you don't have to.